m4_sessions/1806-inning/group1.md

```diff
@@ -1,33 +1,37 @@
 ## Group Members:
-
-
+Adam Conway, Seamus Quinn, Cameron Buscher, Eric Jungbluth
 
 ## Name of Session: 
+You're not an Imposter: You belong to the light, you belong to the thunder.
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Think you're not a developer? You're not the only one! Learn about imposter syndrome and how to break the cycle.
 
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
+Normalize imposter syndrome
+Learn more about it
+Learn to break the cycle
+
 
 ## Session Outline 
 
 (Please include who is facilitating and how much time you are going to spend in each section):
+Introductions : 10 mins with full group
+
+Seamus personal story/
+Small groups sharing their ideas on what it means to them/what they've experienced : 20mins with Seamus
 
+Cameron personal story/
+Teaching about imposter cycle/5 types of imposter syndrome
+Eric personal story/ : 20mins with Cam and Eric
 
+Adam personal story
+Coping with imposter syndrome/How to interrupt the cycle
+Wrap it up/closing discussion : 20mins with Adam
```

Comment:
Love the personal stories and that you are moving through different parts of the cycle.  I'd like you to...

1) Add a few slides or a walk through of "What is imposter syndrome" and include some of the facts/research about it
2) There are HUNDREDS of articles on this, start throwing some into your slack channel and think about ways for students to access these
3) Would love to see a slide/resources or the CONCRETE steps on how to interrupt the cycle